moped-database/migrations/1654057324421_alter_table_public_moped_project_files_alter_column_is_retired/up.sql

```diff
@@ -0,0 +1,2 @@
+COMMENT ON COLUMN "public"."moped_project_files"."is_retired" IS E'Indicates soft deletion';
```

Comment:
Your use of the comment embedded in the database schema here is so good. I support this pattern, and I think we should keep doing it whenever possible. ✅